### ISSUE TITLE
Produces an error when offline

### ISSUE THREAD

**Issue by fronteed** [shanalikhan/code-settings-sync#97]
**Visual Studio Code Version :** 1.5.1
**Code Sync Settings Version :** 2.2.3
**Operating System :** OS X (El Capitan)

@shanalikhan thanks for the nice plugin!

Every time I launch VS Code when I'm offline, I get this error:

```
2//Applications/Visual Studio Code.app/Contents/Resources/app/out/vs/workbench/workbench.main.js:118 [Extension Host] Error: getaddrinfo ENOTFOUND api.github.com api.github.com:443
    at global.sendError (/Users/fronteed/.vscode/extensions/Shan.code-settings-sync-2.2.3/node_modules/github/lib/index.js:783:19)
    at /Users/fronteed/.vscode/extensions/Shan.code-settings-sync-2.2.3/node_modules/github/lib/index.js:792:29
    at callCallback (/Users/fronteed/.vscode/extensions/Shan.code-settings-sync-2.2.3/node_modules/github/lib/index.js:667:17)
    at ClientRequest.<anonymous> (/Users/fronteed/.vscode/extensions/Shan.code-settings-sync-2.2.3/node_modules/github/lib/index.js:739:17)
    at emitOne (events.js:90:13)
    at ClientRequest.emit (events.js:182:7)
    at TLSSocket.socketErrorListener (_http_client.js:295:9)
    at emitOne (events.js:90:13)
    at TLSSocket.emit (events.js:182:7)
    at connectErrorNT (net.js:996:8)
    at _combinedTickCallback (internal/process/next_tick.js:74:11)
    at process._tickCallback (internal/process/next_tick.js:98:9)(anonymous function) @ //Applications/Visual Studio Code.app/Contents/Resources/app/out/vs/workbench/workbench.main.js:118
//Applications/Visual Studio Code.app/Contents/Resources/app/out/vs/workbench/workbench.main.js:93 Error Logged In Console (Help menu > Toggle Developer Tools). You may open an issue using 'Sync : Open Issue' from advance setting command.
```

![2016-09-10-do29k](https://cloud.githubusercontent.com/assets/1612675/18410375/029650b6-776a-11e6-9bdf-a956a2c38ccf.png)
Can it be fixed, please?

**Comment by shanalikhan** [comment]
I have pushed an update 2.2.6 
Can you check and let me know

**Comment by fronteed** [comment]
Error is successfully disappeared, thanks!

**Comment by shanalikhan** [comment]
re-opened! i will fix this soon again :)

**Comment by shanalikhan** [comment]
its fixed in 2.3.3
